src/pages/components/dropdown/style.mdx

```diff
@@ -149,13 +149,37 @@ structure, and size and AI presence.
 
 <Tab label="Default">
 
-![Default multiselect dropdown states](images/dropdown-multiselect-style-default-states.png)
+![Default combo box dropdown states](images/combobox-style-default-states.png)
```

Comment:
```suggestion
![Default combo box interactive states](images/combobox-style-default-states.png)
```

We typically don't refer to it as "combo box dropdown" and instead we just say "combo box" to not confuse people with the "dropdown" variant.